app/services/service_catalog/submit_order.rb

```diff
@@ -0,0 +1,48 @@
+module ServiceCatalog
+  class SubmitOrder
+    attr_reader :order
+
+    def initialize(order_id)
+      @order_id = order_id
+    end
+
+    def process
+      @order = Order.find_by!(:id => @order_id)
+      @order.order_items.each do |order_item|
+        submit_order_item(order_item)
+      end
+      @order.update(:state => 'Ordered', :ordered_at => Time.now.utc)
```

Comment:
I'm wondering if all of this should be in a transaction.

Reply:
@mkanoor Let's investigate putting this in a transaction in a follow up PR, and/or handling weird things like partial orders (first order item succeeds but second order item fails, and the order is never marked as ordered.